### ISSUE TITLE
Finetuning using standard hugging face training code

### ISSUE THREAD

**Issue by urstrulyajay** [tatsu-lab/stanford_alpaca#23]
In ReadMe.md I saw that the model is finetuned using Stanford hugging face setup. I tried it but getting this error. Could someone help in calling Llama weights using hf

from transformers import AutoModelForCausalLM

model = AutoModelForCausalLM.from_pretrained("Bitsy/llama-7b-hfcompatible-clean")

Error : 

KeyError                                  Traceback (most recent call last)
[<ipython-input-3-7730437af4f4>](https://localhost:8080/#) in <module>
      1 from transformers import AutoModelForCausalLM
      2 
----> 3 model = AutoModelForCausalLM.from_pretrained("Bitsy/llama-7b-hfcompatible-clean")

2 frames
[/usr/local/lib/python3.9/dist-packages/transformers/models/auto/configuration_auto.py](https://localhost:8080/#) in __getitem__(self, key)
    577             return self._extra_content[key]
    578         if key not in self._mapping:
--> 579             raise KeyError(key)
    580         value = self._mapping[key]
    581         module_name = model_type_to_module_name(key)

KeyError: 'llama'

**Comment by lxuechen** [comment]
Thanks for your interest. We will release the training pipeline once llama is merged into huggingface main branch.

**Comment by YannDubs** [comment]
@urstrulyajay see https://github.com/tatsu-lab/stanford_alpaca/tree/main#fine-tuning
